src/features/smartSearch/components/types.ts

```diff
@@ -119,14 +119,17 @@ export interface MostActiveFilterConfig {
   size: number;
 }
 
+export type Gender = 'f' | 'm' | 'o' | 'unknown';
```

Comment:
I'll make the changes and comment out what is related to `unknown`!

Reply:
Just to clarify: Once you've made the changes, I think it's enough to comment/filter out whatever makes the option selectable for the user, so that should only have to be in one place. Hopefully you don't have to comment out lots of code.

Reply:
Now I think I am misunderstanding this... I'll leave it until we can talk and I can make sure I understand